crates/bevy_ecs/src/reflect/map_entities.rs

```diff
@@ -13,8 +13,8 @@ use bevy_utils::EntityHashMap;
 /// See [`MapEntities`] for more information.
```

Comment:
I think we should link to `SceneEntityMapper` since it's what it uses under the hood.
```suggestion
/// See [`SceneEntityMapper`] for more information.
```

Reply:
i'll link both